aks/extra-variables.tf

```
@@ -1,8 +1,14 @@
 variable "metrics_storage" {
-  description = "Azure Blob Storage configuration values for the storage container where the archived metrics will be stored."
+  description = "Azure metrics storage configuration"
```

I agree I maybe did a really verbose description last time around, but I think we should not be so vague and at least contain the type of storage that this variable expects. Something like:

```suggestion
  description = "Azure Blob Storage configuration for metric archival."
```

I'll approve the PR regardless, this is just a suggestion.